bigtable/google/cloud/bigtable/instance.py

```diff
@@ -241,23 +241,37 @@ def create(self, location_id=_EXISTING_INSTANCE_LOCATION_ID,
             clusters=clusters)
 
     def update(self):
-        """Update this instance.
+        """Partially updates an instance within a project.
 
         .. note::
 
-            Updates the ``display_name``. To change that value before
-            updating, reset its values via
+            Apdates any or all of the following values:
+            ``display_name``
+            ``type``
+            ``labels``
+            To change a value before
+            updating, reset that values via
 
             .. code:: python
 
                 instance.display_name = 'New display name'
 
             before calling :meth:`update`.
         """
-        self._client.instance_admin_client.update_instance(
+        update_mask = field_mask_pb2.FieldMask()
+        if self.display_name is not None:
+            update_mask.paths.append('display_name')
+        if self.type_ is not None:
+            update_mask.paths.append('type')
+        if self.labels is not None:
+            update_mask.paths.append('labels')
+        update_instance = instance_pb2.Instance(
```

Comment:
Please rename `update_instance` to `nstance_pb`

Reply:
changed to `instance_pb`, also changed `update_mask` to `update_mask_pb`